Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -43,8 +42,10 @@
 
         let checkInfo3 = CheckInfo(id: "id3", hint: "hint3", severity: .error)
         Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)],
-            in: CheckInfo(id: "id3", hint: "hint3", severity: .error)
+            violations: [
+                CheckInfo(id: "id3", hint: "hint3", severity: .error):
+                [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)